TwitchLib.Api/Exceptions/TooManyRequestsException.cs

```diff
@@ -4,13 +4,28 @@ namespace TwitchLib.Api.Exceptions
 {
     /// <inheritdoc />
     /// <summary>Exception representing a 429 Http Statuscode</summary>
-    public class TooManyRequestsException : Exception
+    public sealed class TooManyRequestsException : Exception
     {
         /// <inheritdoc />
         /// <summary>Exception constructor</summary>
         public TooManyRequestsException(string data)
```

Comment:
This constructor is used by the TwitchWebRequest class, however, that actually means the TwitchWebRequest class isn't updated yet to support the new constructor. So I'd remove this constructor and update the TwitchWebRequest class to also support adding a resetTime.

Reply:
Great catch

Reply:
Done!